FluentUI.Demo/src/main/res/layout/activity_vnext_basic_inputs.xml

```diff
@@ -0,0 +1,232 @@
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:divider="@drawable/demo_divider"
+    android:orientation="vertical"
+    android:padding="@dimen/default_layout_margin"
+    android:showDividers="middle">
+
+    <TextView
+        style="@style/TextAppearance.FluentUI.Headline"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingBottom="@dimen/demo_headline_padding_bottom"
+        android:text="@string/button" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/demo_headline_divider_height"
+        android:layout_marginBottom="@dimen/default_layout_margin"
+        android:background="@drawable/ms_row_divider" />
+
+    <!--For the basic button, style can be set in the app theme.-->
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <com.microsoft.fluentui.vNext.Button
+            android:id="@+id/primary_button_small"
+            style="@style/Widget.FluentUI.primarybuttontokensview.small"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/button" />
+
+        <com.microsoft.fluentui.vNext.Button
+            android:id="@+id/primary_button_small_disabled"
+            style="@style/Widget.FluentUI.primarybuttontokensview.small"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="30dp"
+            android:enabled="false"
+            android:text="@string/button_disabled" />
+
+    </LinearLayout>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:orientation="horizontal">
+
+        <com.microsoft.fluentui.vNext.Button
+            android:id="@+id/primary_button"
+            style="@style/Widget.FluentUI.primarybuttontokensview.medium"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/button" />
+
+        <com.microsoft.fluentui.vNext.Button
+            style="@style/Widget.FluentUI.primarybuttontokensview.medium"
+            android:id="@+id/primary_button_disabled"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="30dp"
+            android:enabled="false"
+            android:text="@string/button_disabled" />
+
+    </LinearLayout>
+    <!--For other button variants, style can be set in the widget.-->
```

Comment:
It was added from the existing xml for buttons.
We can set basic style in the app theme which can be used across the app and for other styles you can specify the style for each button widget
For example:- if we set primary button in the app theme, then all buttons will have that style
but if you need some other style like secondary button then you need to specify that in the button widget directly.

In this xml, we cab remove these comments because we arre not setting app theme with vnext button style and hence for all button widget, the style needs to be explicitly mentioned
Will remove the comments